### PR TITLE
pipenv

### PR DESCRIPTION
**Pull request by golan13**
related to issue #28 
Migrated app to use Pipenv
Translated requirements.txt file to Pipfile and Pipfile.lock
Edited bootstrap file to install Pipenv and apply the correct env

### CODE REVIEW

**Review by mosesnagar — APPROVED** [review]
As far as I read about it, it looks good.
Maybe you can add on CONTRIBUTING.md about how to add packages and how to use python command on `vagrant ssh` using the virtual env.

**Comment by golan13** [comment]
@mosesnagar Good idea. Will do

**Review by lioramilbaum — APPROVED** [review]
Please link the PR to the issue it resolves. It enables traceability which is valuable for future trouble shooting/debugging. And, GitHub closes the issue automatically when the PR is merged.

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
```
Comment (ifireball) [inline Pipfile:8]: did you convert this file for a previous file? Generally `Pipfile` should not be this detailed - that is the role of ` Pipfile.lock`.

the `Pipfile` sould generally list only the 1st level dependencies for your project and avoid listing exact versions so you can use `pipenv update` to automate updating dependencies.

File: bootstrap.sh [hunk 2]
```diff
@@ -8,7 +8,9 @@ apt install -y python3.7 python3-pip postgresql postgresql-contrib
 
 echo "installing dependencies"
 pip3 install --upgrade pip
```
Comment (ifireball) [inline bootstrap.sh:10]: since you're using pipenv I expect this line to no longer be needed.

File: bootstrap.sh [hunk 3]
```diff
@@ -8,7 +8,9 @@ apt install -y python3.7 python3-pip postgresql postgresql-contrib
...
-pip3 install -r /vagrant/requirements.txt
+pip3 install pipenv
```
Comment (ifireball) [inline bootstrap.sh:11]: consider adding `--user` to avoid touching the system python.

File: bootstrap.sh [hunk 4]
```diff
@@ -8,7 +8,9 @@ apt install -y python3.7 python3-pip postgresql postgresql-contrib
...
+export PIPENV_VENV_IN_PROJECT=1
```
Comment (ifireball) [inline bootstrap.sh:12]: I don't think you really need this - if you do, add a comment why, since this is not the default way of using pipenv.

**Review by ifireball — CHANGES_REQUESTED** [review]
The PR title is misleading - you're adding `pipenv` not `virtualenv`. Also added some comments about the code.

File: bootstrap.sh (hunk 4 above)
Comment (golan13) [reply bootstrap.sh:12]: When I removed this line, the virtual environment is always created under root user. After I try to SSH into the machine the logged in user is vagrant and so I couldn't access the virtual environment.

File: Pipfile (hunk 1 above)
Comment (golan13) [reply Pipfile:8]: Yes it's converted from the requirements.txt file that we had before

**Pull request closed without merging**